src/components/ConversionProcessingOverlay/index.tsx

```diff
@@ -29,40 +29,39 @@ const ConversionProcessingOverlay = ({
                     cancelProcessing={toggleProcessing}
                 />
             ) : null}
-            <h2 className={styles.titleText}> File conversion in progress </h2>
-            <p className={styles.centerText}>
+            <h2 className={styles.title}> File conversion in progress </h2>
+            <div className={styles.content}>
```

Comment:
`title` is as clear as `titleText`, `content` is less confusing than `centerText` (which seems like you're trying to center the text)

Reply:
I grouped everything within this container because conceptually they go together